src/dags/importscripts/import_risicozones.py

```diff
@@ -0,0 +1,393 @@
+import ccsv as csv
+import json
+import sys
+import hashlib
+from collections import namedtuple
```

Comment:
Je maakt gebruik van de namedtuple. Die is vooral handig om velden direct als attributen te kunnen gebruiken. Maar, in de code wordt de namedtuple nergens op die manier gebruikt, maar worden de attributen altijd dynamisch gehouden. Kun je in dat geval niet beter gewone dictionaries gebruiken? Want dan zit de namedtuple eigenlijk alleen maar in de weg.

Reply:
Ja, ik zie het nu ook. Was handig in het begin om de data goed te kunnen analyseren. Maar nu niet meer nodig. Ik haal het weg en gebruik dan gewone dictionaries, wordt dan ook een stukje overzichtelijker